x-pack/plugins/lists/server/scripts/quick_start.sh

```diff
@@ -0,0 +1,11 @@
+./hard_reset.sh && \
+./post_list.sh lists/new/lists/keyword.json && \
+./post_list_item.sh lists/new/list_keyword_item.json && \
+./post_exception_list.sh && \
+./post_exception_list_item.sh ./exception_lists/new/exception_list_item_with_list.json
+# ./post_exception_list.sh
+# ./post_exception_list_item.sh
+#./post_exception_list_item.sh ./exception_lists/new/exception_list_item_with_list.json
+#./post_list.sh lists/new/list_keyword.json
+#./post_list_item.sh lists/new/list_keyword_item.json
```

Comment:
Remove the commented out stuff unless you really want it or add some comments about when/why you would uncomment them?

Reply:
Yeah I think I should just remove the commented out pieces.